src/main/webapp/i18n/de/exam.json

```diff
@@ -313,98 +315,60 @@
-                "exerciseGroups": "Du kannst Übungsgruppen mit mehreren Übungsvarianten erstellen, um Unterschleif für Studierende zu erschweren. Die Punkte aller Aufgaben einer Aufgabengruppe müssen identisch sein. Die Anzahl an Aufgabengruppen muss der Anzahl der Aufgaben in der Klausur entsprechen.",
-                "generateStudentExams": "Die Klausuren der Studierenden müssen generiert worden sein, bevor die Klausur beginnt. Die Arbeitszeit kann dabei individuell angepasst werden.\nZudem werden alle Commits des Templates von Programmieraufgaben in einem Commit zusammengefasst.",
-                "prepareExerciseStart": "Vorbereiten der Aufgaben kann die Serverlast während der Klausur reduzieren.",
-                "examDetails": "Alle relevanten Details der Klausur müssen vor dem Start der Korrektur eingetragen sein.",
-                "conductExam": "In der Studierenden-Klausurübersicht sieht man, welche Studierende ihre Klausur bereits abgegeben haben.",
-                "assessAllSubmissions": "Die Anzahl der Korrekturrunden der Aufgaben entspricht der in der Klausur angegebenen Anzahl von Korrekturrunden.",
-                "publishResults": "Das Veröffentlichen der Ergebnisse macht diese zugänglich für die Teilnehmenden der Klausur.",
-                "examReview": "Alle relevanten Daten müssen in der Klausur spezifiziert sein.",
-                "resolveComplaints": "Beschwerden können im Bewertungs-Dashboard bearbeitet werden.",
-                "testRun": "Probiere verschiedene Kombinationen von Übungsvarianten aus Sicht der Studierenden aus.",
-                "exportResults": "Die Klausurergebnisse können in Form einer .csv Datei exportiert werden."
+            "delete": {
+                "question": "Soll die Klausur <strong>{{ title }}</strong> wirklich dauerhaft gelöscht werden? Alle zugehörigen Elemente werden gelöscht, inklusive der Klausuren der Studierenden. Diese Aktion kann NICHT rückgängig gemacht werden!",
+                "typeNameToConfirm": "Bitte gib den Namen der Klausur zur Bestätigung ein."
             },
-
             "reset": {
                 "question": "Soll die Klausur <strong>{{ title }}</strong> wirklich dauerhaft zurückgesetzt werden? Alle Klausuren der Studierenden werden gelöscht. Teilnehmende an der Klausur und die Konfigurationen der Aufgaben bleiben erhalten. Diese Aktion kann NICHT rückgängig gemacht werden!",
                 "typeNameToConfirm": "Bitte gib den Namen der Klausur zur Bestätigung ein."
             },
             "checklist": {
                 "title": "Checkliste für die Durchführung von Prüfungen",
-                "introduction": "Unter folgendem Link kannst du dich über den Ablauf einer Artemis Klausur informieren:",
+                "introduction": "Unter folgendem Link kannst du dich über den Ablauf einer Klausur in Artemis informieren:",
```

Comment:
```suggestion
                "introduction": "Unter folgendem Link kannst du dich über den Ablauf einer Klausur auf Artemis informieren:",
```

Reply:
The feature overview page uses both ‘auf’ and ‘in’ for ‘Onlineklausuren in/auf Artemis’. I personally favour ‘in’, but also don’t want to just dismiss your suggestion as both variants are perfectly valid ways to phrase this.